src/test/resources/fixtures/hmh-ri/caliperEventGradeGraded.json

```diff
@@ -0,0 +1,65 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "id":"urn:uuid:uniqueIdPerEvent",
+  "type": "GradeEvent",
+  "actor": {
+    "id": "https://url/to/lexileService",
+    "type": "SoftwareApplication",
+    "version": "v1"
+  },
+  "action": "Graded",
+  "object": {
+    "id": "https://www.hmhco.com/readinginventory/quiz/c35603f7a3434e2cb74ef14d60e70f42/instance/belonging/to/student/userRefId/item/itemId1/attempts/completedAttemptId",
+    "type": "Attempt",
+    "assignee": {
+      "id":"https://https://www.hmhco.com/ed/users/userRefId",
+      "type": "Person",
+      "name" : "Casandra Rath"
+    },
+    "assignable": {
+      "id":"https://www.hmhco.com/readinginventory/quiz/c35603f7a3434e2cb74ef14d60e70f42/instance/belonging/to/student/userRefId",
+      "type": "Assessment"
+    },
+    "count": 100, (count of all the students attempts made during the whole assignment)
+    "dateCreated": "2016-11-15T10:05:00.000Z",
+    "startedAtTime": "2016-11-15T10:05:00.000Z",
+    "endedAtTime": "2016-11-15T10:55:12.000Z"
+  },
+  "eventTime": "2016-11-15T10:57:06.000Z",
+  "edApp":{
+    "id":"https://www.hmhco.com/readinginventory",
+    "type":"SoftwareApplication",
+    "version":"v1"
+  },
+  "generated": {
+    "id": "https://www.hmhco.com/readinginventory/quiz/c35603f7a3434e2cb74ef14d60e70f42/instance/belonging/to/student/userRefId/attempts/completedAttemptId/scores/1",
+    "type": "Score",
+    "attempt": "https://www.hmhco.com/readinginventory/quiz/c35603f7a3434e2cb74ef14d60e70f42/instance/belonging/to/student/userRefId/attempts/completedAttemptId",
+    "maxScore": 15.0,
+    "scoreGiven": -487.0,
+    "scoredBy": "https://url/to/lexileService", (actor id)
+    "comment": "The lexile score from Test Quiz # c35603f7a3434e2cb74ef14d60e70f42 taken on date/time for student userRefId",
+    "dateCreated": "2016-11-15T10:56:00.000Z"
+  },
+  "group": {
+    "id": "https://url/to/ids/district/school/sections/sectionId1",
+    "type": "CourseSection",
+    "courseNumber": "CPS 435-01",
+    "academicSession": "Fall 2016"
+  },
+  "membership":{
```

Comment:
@padraig-ohiceadha should I omit the group and membership because the actor is the lexile service?